content/blog/getting-started-with-spark-on-mapr-sandbox.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started with Spark on MapR Sandbox
+date: 2021-10-14T06:01:00.000Z
+author: MapR Tutorials
+authorimage: /img/Avatar1.svg
+---
+
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+At MapR, we distribute and support Apache Spark as part of the MapR Converged Data Platform. This tutorial will help you get started with running Spark applications on the MapR Sandbox.
```

Comment:
Nathan,  last year the Mapr Sandbox  was one of the pending migration items from MapR.com. 

The link below mentioned in the next paragraph points to the Development Environment for HPE Ezmeral DF:
 https://docs.datafabric.hpe.com/62/MapRContainerDevelopers/RunMapRContainerDevelopers.html.

Do you confirm that the **MapR Sandbox** is no longer a pending migration item from MapR.com and now the "Development Environment for HPE Ezmeral Data Fabric"?

If it is confirmed, I suggest you add a note in this sentence to something like this:

This tutorial will help you get started with running Spark application on the MapR Sandbox (now known as the Development Environment for HPE Ezmeral Data Fabric).

What do you think?

Reply:
done